### PR TITLE
Rename method arguments to use camelCase casing (libraries/cms directory)

### PR DESCRIPTION
**Pull request by SharkyKZ**
Partial Pull Request for Issue https://github.com/joomla/joomla-cms/issues/30988.

### Summary of Changes

Renames some method arguments to use camelCase casing.

### Testing Instructions

Thorough code review / Joomla works like before.

### Documentation Changes Required

No.

### CODE REVIEW

File: libraries/cms/html/jgrid.php [hunk 1]
```diff
@@ -21,34 +21,34 @@ abstract class JHtmlJGrid
...
+	 * @param   string|array  $prefix         An optional task prefix or an array of options
+	 * @param   string        $text           An optional text to display [unused - @deprecated 4.0]
+	 * @param   string        $activeTitle    An optional active tooltip to display if $enable is true
+	 * @param   string        $inactiveTitle  An optional inactive tooltip to display if $enable is true
+	 * @param   boolean       $tip            An optional setting for tooltip
+	 * @param   string        $activeClass    An optional active HTML class
+	 * @param   string        $inactiveClass  An optional inactive HTML class
+	 * @param   boolean       $enabled        An optional setting for access control on the action.
+	 * @param   boolean       $translate      An optional setting for translation.
+	 * @param   string        $checkbox	      An optional prefix for checkboxes.
```
Comment (Quy) [inline libraries/cms/html/jgrid.php]:
```suggestion
	 * @param   string        $checkbox	  An optional prefix for checkboxes.
```

File: libraries/cms/html/jgrid.php [hunk 2]
```diff
@@ -21,34 +21,34 @@ abstract class JHtmlJGrid
...
+	 * @param   string|array  $prefix         An optional task prefix or an array of options
+	 * @param   string        $text           An optional text to display [unused - @deprecated 4.0]
+	 * @param   string        $activeTitle    An optional active tooltip to display if $enable is true
+	 * @param   string        $inactiveTitle  An optional inactive tooltip to display if $enable is true
+	 * @param   boolean       $tip            An optional setting for tooltip
+	 * @param   string        $activeClass    An optional active HTML class
+	 * @param   string        $inactiveClass  An optional inactive HTML class
+	 * @param   boolean       $enabled        An optional setting for access control on the action.
+	 * @param   boolean       $translate      An optional setting for translation.
+	 * @param   string        $checkbox	  An optional prefix for checkboxes.
```
Comment (zero-24) [inline libraries/cms/html/jgrid.php]:
```suggestion
	 * @param   string        $checkbox       An optional prefix for checkboxes.
```

Comment (SharkyKZ) [reply libraries/cms/html/jgrid.php]: It was there before but OK.

**Comment by Quy** [comment]
I have tested this item :white_check_mark: successfully on 995e9f224d25351026bb5ea39dd8d35addc95b37<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/31149">issues.joomla.org/tracker/joomla-cms/31149</a>.</sub>

**Comment by alikon** [comment]
I have tested this item :white_check_mark: successfully on 995e9f224d25351026bb5ea39dd8d35addc95b37<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/31149">issues.joomla.org/tracker/joomla-cms/31149</a>.</sub>

**Comment by alikon** [comment]
RTC<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/31149">issues.joomla.org/tracker/joomla-cms/31149</a>.</sub>

**Pull request merged**